src/v/cloud_storage/recursive_directory_walker.cc

```diff
@@ -42,10 +42,14 @@ struct walk_accumulator {
         seen_dentries = true;
         auto entry_path = fmt::format("{}/{}", target, entry.name);
         if (entry.type && entry.type == ss::directory_entry_type::regular) {
-            vlog(cst_log.debug, "Regular file found {}", entry_path);
-
             auto file_stats = co_await ss::file_stat(entry_path);
 
+            vlog(
+              cst_log.debug,
+              "Regular file found {} ({})",
```

Comment:
nit: an indication that the number in parens is a size, like a unit, would be helpful.